### PR TITLE
[docs] Add docs in latex format from OSTIS Standard

### PR DESCRIPTION
**Pull request by NikitaZotov**
* [x] Read PR [documentation](https://github.com/ostis-ai/sc-machine/blob/main/docs/dev/pr.md/)
* [x] Update changelog
* [x] Update documentation

Improvement of sc-machine documentation. Changes:
- Add latex docs from OSTIS Standard;
- Unite mkdocs and latex docs.

This PR depends on #72.

### CODE REVIEW

File: docs/sc-network/sc_network.tex [hunk 1]
```diff
@@ -0,0 +1,12 @@
+\scnheader{Реализация подсистемы взаимодействия с внешней средой с использованием сетевых протоколов}
+\scnrelfromlist{компонент программной системы}{РРеализация подсистемы взаимодействия с внешней средой с использованием протоколов на основе формата JSON}
```
Comment (MikhailSadovsky) [inline docs/sc-network/sc_network.tex:2]:
```suggestion
\scnrelfromlist{компонент программной системы}{Реализация подсистемы взаимодействия с внешней средой с использованием протоколов на основе формата JSON}
```

File: docs/sc-control/sc_lock.tex [hunk 2]
```diff
@@ -0,0 +1,4 @@
+\scnheader{блокировка sc-элемента в рамках программной модели sc-памяти}
+\scnidtf{ScLock}
+\scnrelto{класс компонентов}{Реализация sc-хранилища}
+\scnrelfrom{смотрите}{\nameref{sd_agents}}
```
Comment (MikhailSadovsky) [inline docs/sc-control/sc_lock.tex:4]: Ссылка не отображается в итоговом pdf-документе

**Review by MikhailSadovsky — CHANGES_REQUESTED** [review]

**Review by MikhailSadovsky — APPROVED** [review]

**Review by KovalM — APPROVED** [review]

**Review by MksmOrlov — CHANGES_REQUESTED** [review]
remove docs/sc-memory/sc-storage/images/sc_storage.pdf

**Comment by NikitaZotov** [comment]
> remove docs/sc-memory/sc-storage/images/sc_storage.pdf

It is an image. It is needed

**Review by MksmOrlov — APPROVED** [review]

**Pull request merged**